src/hooks/useDisplayedWallets.ts

```diff
@@ -7,6 +7,8 @@ export const useDisplayedWallets = () => {
   const displayedWallets = [
     WalletType.MetaMask,
 
+    (isTestnet || isDev) && WalletType.Phantom,
```

Comment:
Why `isTestnet` and not just `isDev`?

Reply:
FWIW it may be better this way, both verecl environments deploy on every PR so we'll be able to check it out in both deployments. but i'm agnostic to this as long as it's gated from prod